### DISCUSSION TITLE
Spotify Lyrics not working

### DISCUSSION THREAD

**Discussion by nnmmaa1234** [SpotX-Official/SpotX#497] [Errors]
Hi! Just wanted to ask that is it a glitch from my pc or is it removed in the most recent update? Because I've tried clearing cache and all, but can't seem to get the lyrics feature back :)

**Comment by amd64fox** [comment]
are there just no lyrics or are they there but they don’t work? show full screenshot of client or video

**Reply by nnmmaa1234** [reply]
![image](https://github.com/amd64fox/SpotX/assets/57182904/d31d83ec-4961-438f-b1ae-d3043ec952d4)
 I used to get lyrics before but now I can't get lyrics

**Reply by amd64fox** [reply]
this song has no lyrics, play [this song](https://open.spotify.com/track/3GZD6HmiNUhxXYf8Gch723?si=5013efa4d97142ac&nd=1) and send the screenshot again

also what is the region of your account?

**Reply by nnmmaa1234** [reply]
![image](https://github.com/amd64fox/SpotX/assets/57182904/84957f61-dc08-4da2-ad18-b22d276f5813)
no w8.... I'm getting lyrics of a few songs... but not all.
like this one 
![image](https://github.com/amd64fox/SpotX/assets/57182904/1e75d79e-2bd6-4d66-9686-4fd19805a7f9)
at first I could get lyrics of all the songs......

does this have to do with spotify update or some bug from spotx?
because i accidentally uninstalled mine .... then this started happening after i reinstalled it.

**Reply by amd64fox** [reply]
Spotify (Musixmatch) can remove or add lyrics for songs at any time, spotx has no effect on this

**Reply by nnmmaa1234** [reply]
ah i see
thank you for clarifying :)

i just got worried because until a few moments ago i had all the lyrics but suddenly they started disappearing after i reinstalled it.
must be because i had an older version installed.

**Reply by amd64fox** [reply]
lyrics are not stored in the client; if they are not available, they won't be on any client, whether it's old or new, mobile or desktop. Actually, there is currently testing of song lyrics for some accounts from India and Italy, where they have completely restricted access to lyrics. However, since you have access to some song lyrics, it didn't affect you

**Reply by hanfclippy** [reply]
You can try installing with the -old_lyrics parameter, worked for me!

**Reply by JDArrOw3** [reply]
> You can try installing with the -old_lyrics parameter, worked for me!

How to do that?

**Reply by hanfclippy** [reply]
Go to Powershell, do ```iex "& { $(iwr -useb 'https://raw.githubusercontent.com/SpotX-Official/spotx-official.github.io/main/run.ps1') } -new_theme -old_lyrics -topsearchbar -adsections_off -podcasts_on -block_update_on"```, this will install completely stock Spotify just without ads/ad sections.

**Comment by EricWhite8452** [comment]
my Spotify lyrics on Windows not showing either.

**Reply by nnmmaa1234** [reply]
most of the songs on spotify... and by that I mean 80% of the songs don't have lyrics, in the previous versions of spotify they did but in the recent update they don't so like amd64fox said.... it's spotify, 
spotX has nothing to do with this. 
hopefully we start getting out lyrics back in the next updates :)

**Comment by ENLYOWH** [comment]
Can you make it possible to pin the lyrics window or canvas so we can read the lyrics while doing something else

**Comment by NewayJr** [comment]
Clearly there's sth wrong with the lyrics. I checked for some songs I had listened to with lyrics a few days before installing this patch, and after, then they disappeared after the patch. 
Still, respect to those who made it possible🏆

**Reply by NewayJr** [reply]
Oh I got it now! They are moved to the side panel right below where you find the information of the artist.😍

**Reply by arizkyp55** [reply]
thanks, i got it.
but why not by icon button mic like the original spotify? hmm but it's okay then. thank you very much for the devs 🙏

**Reply by WildDV** [reply]
If you change the parameter in the installation it works
![Spo](https://github.com/SpotX-Official/SpotX/assets/123776174/73fdb827-b1db-4a7a-8b07-a236f1a51e61)
![spo2](https://github.com/SpotX-Official/SpotX/assets/123776174/af7705a1-21c8-4c22-aca7-2345bfd897bf)

**Comment by hanfclippy** [comment]
> ah i see thank you for clarifying :)
> 
> i just got worried because until a few moments ago i had all the lyrics but suddenly they started disappearing after i reinstalled it. must be because i had an older version installed.



> thanks, i got it. but why not by icon button mic like the original spotify? hmm but it's okay then. thank you very much for the devs 🙏

You can install with the -old_lyrics parameter, then you get that button back!

**Reply by RepublicofMar** [reply]
Hi ! newbie here. Can you give a tutorial how to make it works. Thanks in advance.

**Reply by WildDV** [reply]
You just have to change this parameter in the installation file.
![Spo](https://github.com/SpotX-Official/SpotX/assets/123776174/8655e547-1580-44e1-8ceb-0093be383093)

**Comment by yodaluca23** [comment]
Don't know if this is related,

Spotify recently implemented a new system where free users can only tap see all lyrics a certain amount of times per month. Will this be able to be circumnavigated with SpotX or no I think worst case scenario we could make it where it would replace the provided lyrics with lyrics directly from a lyrics service...

**Reply by yodaluca23** [reply]
Basically now it’s servers sided and they changed a bunch of stuff free users only get like three full lyric searches a month. More information in these issues for this Spotify iOS Mod, they also implemented it where if you reached that limit, it will get the lyrics from Genius automatically and replace them. 

https://github.com/whoeevee/EeveeSpotify/issues/42
https://github.com/whoeevee/EeveeSpotify/issues/43

**Comment by GiorgosSid** [comment]
Are all the lyrics now disabled in SpotX. All the songs even the famous ones have only a fraction of the total lyrics they have. I cant find anything to fix this issue
![Screenshot 2024-05-06 114438](https://github.com/SpotX-Official/SpotX/assets/72700053/f8506254-9a29-46bb-bf53-46fcc5083a2f)

**Reply by Spinoscythe** [reply]
this is a Spotify cash grab dick move

**Reply by ash-byee** [reply]
Yeah, I just reinstalled the spotx patch with a few parameters and noticed that it said the same thing for me. I was trying to fix it with the "-old_lyrics" but that didn't work either. Is this an issue impacting only spotx users or all non-premium users?

**Reply by MaybeMarcel** [reply]
The problem is for all free users. If you find a key to fix it tell me pls.

**Reply by Kiogra** [reply]
Idem here, please update me if they found a fix

**Comment by wilhelm115** [comment]
Wouldn't it be easier to just grab the lyrics from https://www.musixmatch.com and print it out ?
No sync, no special format, just linking a iframe to the "Playing View" ?

**Comment by RapZ2** [comment]
![image](https://github.com/SpotX-Official/SpotX/assets/169634015/b8e478eb-aacd-4dc7-8a0a-a05411e9abcc)

anyone know how to fix "lyrics arent synced to the song yet." 
all music like that

**Reply by arizkyp55** [reply]
adding spicetify patch. there is lyrics plus

**Reply by RapZ2** [reply]
can you explain to me how to do it

**Reply by arizkyp55** [reply]
you can do through this https://github.com/SpotX-Official/SpotX/issues/596#issue-2284099888

**Reply by RapZ2** [reply]
how i can add lyrics plus to spotx? Is it in the parameters I added "-lyrics_block" ??

**Reply by arizkyp55** [reply]
no. parameter -lyrics_block is working for blocking native spotify lyrics feature.

meanwhile, lyrics-plus is a feature on spicetify. so you have to install spicetify first with same steps as spotx installation (using PowerShell CLI). Spicetify works linear inside SpotX. You can read more in its official website: https://spicetify.app/

**Reply by RapZ2** [reply]
iwr -useb https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install.ps1 | iex

this command?

**Reply by RepublicofMar** [reply]
![Screenshot (71)](https://github.com/SpotX-Official/SpotX/assets/166490677/bdde41b1-35ea-4555-97e5-9c82dc952e86)

Thanks to @amd64fox and the others for helping everyone ! Everything is fine now. I got the lyrics back and other good features. God bless yall.

**Comment by RapZ2** [comment]
[image: image.png]
look same bro

Pada Sen, 22 Jul 2024 pukul 13.43 BlueSchnabeltier ***@***.***>
menulis:

**Comment by amd64fox** [comment]
topic is frozen to avoid spreading incorrect information

[current info on lyrics](https://github.com/SpotX-Official/SpotX/issues/596)
